modules/cdc-ext/src/main/java/org/apache/ignite/cdc/metrics/KafkaToIgniteMetrics.java

```diff
@@ -0,0 +1,214 @@
+import org.apache.ignite.internal.processors.metric.MetricRegistryImpl;
+import org.apache.ignite.internal.processors.metric.impl.AtomicLongMetric;
+import org.apache.ignite.internal.util.typedef.F;
+import org.apache.ignite.internal.util.typedef.internal.U;
+import org.apache.ignite.spi.IgniteSpi;
+import org.apache.ignite.spi.metric.jmx.JmxMetricExporterSpi;
+import org.apache.ignite.spi.metric.noop.NoopMetricExporterSpi;
+
+import static org.apache.ignite.internal.IgnitionEx.initializeDefaultMBeanServer;
+import static org.apache.ignite.internal.processors.cache.persistence.wal.reader.StandaloneGridKernalContext.closeAllComponents;
+import static org.apache.ignite.internal.processors.cache.persistence.wal.reader.StandaloneGridKernalContext.startAllComponents;
+import static org.apache.ignite.internal.processors.metric.impl.MetricUtils.metricName;
+
+/** CDC kafka to ignite metrics. */
+public class KafkaToIgniteMetrics {
+    /** Count of events received name. */
+    public static final String K2I_EVTS_RSVD_CNT = "EventsReceivedCount";
+
+    /** Count of events received description. */
+    public static final String K2I_EVTS_RSVD_CNT_DESC = "Count of events received from kafka";
+
+    /** Timestamp of last received event name. */
+    public static final String K2I_LAST_EVT_RSVD_TIME = "LastEventReceivedTime";
+
+    /** Timestamp of last received event description. */
+    public static final String K2I_LAST_EVT_RSVD_TIME_DESC = "Timestamp of last received event from kafka";
+
+    /** Count of metadata markers received name. */
+    public static final String K2I_MARKERS_RSVD_CNT = "MarkersCount";
+
+    /** Count of metadata markers received description. */
+    public static final String K2I_MARKERS_RSVD_CNT_DESC = "Count of metadata markers received from Kafka";
+
+    /** Count of events sent name. */
+    public static final String K2I_MSGS_SNT_CNT = "EventsSentCount";
+
+    /** Count of events sent description. */
+    public static final String K2I_MSGS_SNT_CNT_DESC = "Count of events sent to destination cluster";
+
+    /** Timestamp of last sent batch name. */
+    public static final String K2I_LAST_MSG_SNT_TIME = "LastBatchSentTime";
+
+    /** Timestamp of last sent batch description. */
+    public static final String K2I_LAST_MSG_SNT_TIME_DESC = "Timestamp of last sent batch to the destination cluster";
+
+    /** Timestamp of last received message. */
+    private AtomicLongMetric lastRcvdEvtTs;
+
+    /** Count of received events. */
+    private AtomicLongMetric evtsRcvdCnt;
+
+    /** Timestamp of last sent message. */
+    private AtomicLongMetric lastSntMsgTs;
+
+    /** Count of sent events. */
+    private AtomicLongMetric evtsSntCnt;
+
+    /** Count of received markers. */
+    private AtomicLongMetric markersCnt;
+
+    /** Standalone kernal context. */
+    private StandaloneGridKernalContext kctx;
+
+    /** CDC metrics registry. */
+    private MetricRegistryImpl mreg;
```

Comment:
Can be local variable

Reply:
Refactored the naming of the directory for metrics registry. After this the comment is not relevant.